astropy/utils/iers/iers.py

```diff
@@ -670,29 +670,27 @@ def open(cls):
                 return cls.iers_table
 
         dl_success = False
-        err_list = []
+        err_msg = ''
 
-        for url in all_urls:
-            try:
-                filename = download_file(url, cache=True)
-            except Exception as err:
-                err_list.append(str(err))
-            else:
-                dl_success = True
-                break
+        try:
+            filename = download_file(all_urls[0], sources=all_urls, cache=True)
+        except Exception as err:
+            err_msg = str(err)
```

Comment:
I think we can just have the content of `if not dl_success` here - the logic here was only needed when multiple files were tried.